docs/openapi/log-level.yaml

```diff
@@ -0,0 +1,67 @@
+openapi: 3.1.0
+info:
+  version: '1.0'
+  title: 'MicroCeph Log level config REST API'
+paths:
+  /1.0/microceph/config/log-level:
+    put:
+      summary: Set log level for MicroCeph
+      description: Allows the client to configure the logging levels for the MicroCeph daemon.
+      operationId: setLogLevel
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema: 
+              $ref: "#/components/schemas/Level"
+      responses:
+        '200':
+          description: Successfully set log-level
+        '500':
+          description: Internal Error while processing the request.
+          content:
+            application/json:
+              schema: 
+                type: object
+                properties:
+                  error_code:
+                    type: integer
+                    description: HTTP Error code
+                  error:
+                    type: string
+                    description: Description of error.
+    get:
+      summary: Get log level for MicroCeph
+      description: Fetches the current logging level of the MicroCeph daemon.
+      operationId: getLogLevel
+      responses:
+        '200':
+          description: Successfully fetched log-level
+          content:
+            application/json:
+              schema: 
+                type: object
+                properties:
+                  metadata:
+                    type: integer
+                    description: Numerical value of the log-level
+                    example: 4
```

Comment:
umm, i get the gist. Will make changes, however, it cannot be made exactly how you've modelled because the response-body documentation should match actual response. Which is something like this.
```
{
    "type": "error",
    "status": "",
    "status_code": 0,
    "operation": "",
    "error_code": 500,
    "error": "Failed to run: ceph config get  abcd: exit status 22 (Error EINVAL: unrecognized entity '')",
    "metadata": null
}
```

Reply:
updated, please take a look.